docs/spec/errors/error.json

```diff
@@ -98,7 +98,7 @@
         "timestamp": {
             "type": "string",
             "description": "Recorded time of the error, UTC based and formatted as YYYY-MM-DDTHH:mm:ss.sssZ",
-            "pattern": "^(\\d{4})-(\\d{2})-(\\d{2})T(\\d{2}):(\\d{2}):(\\d{2})(\\.\\d{1,6})?Z"
+            "pattern": "^(\\d{4})-(\\d{2})-(\\d{2})T(\\d{2}):(\\d{2}):(\\d{2})(\\.\\d{1,6})?Z$"
```

Comment:
@simitt Is there something like a concept of "variable" in json schema so we could reuse the same pattern for all timestamps instead of copy / pasting it?